DigitalLearningSolutions.Web/ViewModels/Common/NumberOfAdministratorsViewModel.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.Common
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using DigitalLearningSolutions.Data.Models;
+    using DigitalLearningSolutions.Data.Models.User;
+
+    public class NumberOfAdministratorsViewModel
+    {
+        public int Admins { get; set; }
+        public int Supervisors { get; set; }
+        public int Trainers { get; set; }
+        public int TrainerSpotsAvailable { get; set; }
+        public int CmsAdministrators { get; set; }
+        public int CmsAdministratorSpotsAvailable { get; set; }
+        public int CmsManagers { get; set; }
+        public int CmsManagerSpotsAvailable { get; set; }
+        public int CcLicences { get; set; }
+        public int CcLicencesAvailable { get; set; }
+
+        public NumberOfAdministratorsViewModel(Centre centreDetails, List<AdminUser> adminUsers)
+        {
+            TrainerSpotsAvailable = centreDetails.Trainers;
+            CmsAdministratorSpotsAvailable = centreDetails.CmsAdministrators;
+            CmsManagerSpotsAvailable = centreDetails.CmsManagers;
+            CcLicencesAvailable = centreDetails.CcLicences;

```

Comment:
We need to support these values being "-1", which means that the centre has no limits on these spots. :)

Reply:
In the case where the values are -1, should we just display the number that are present? E.g. instead of "X/Y" we just write "X"?

Reply:
Yes please! :)